website/docs/reference/resource-configs/hard-deletes.md

```diff
@@ -0,0 +1,111 @@
+---
+title: hard_deletes
+resource_types: [snapshots]
+description: "Use the `hard_deletes` config to control how deleted rows are tracked in your snapshot table."
+datatype: "{<dictionary>}"
```

Comment:
I'm not sure how the `datatype` is used here (or what the format is supposed to be), but `hard_deletes` it is a boolean rather than a dictionary:

```suggestion
datatype: "boolean"
```